### DISCUSSION TITLE
subprocess name in volumen mixer

### DISCUSSION THREAD

**Discussion by wnunezc** [cefsharp/CefSharp#3602] [Q&A]
When implementing cefsharp in a winform desktop application, and when entering a url it has audio, in the volume mixer it appears as follows; How do you prevent it from appearing with that name and having another name, that of the application or the Url open at the moment?

![image](https://user-images.githubusercontent.com/1207306/120899118-b2a7d380-c5f3-11eb-8a24-0c3b75cba8a2.png)

**Comment by amaitland** [comment]
You will need to self host the browser sub process. Basically use your own application exe as the sub process. 

Instructions at https://github.com/cefsharp/CefSharp/issues/3407#issuecomment-782828504
You can ignore the .Net Core parts, the fundamentals are the same regardless of .Net version.
